### PR TITLE
Add Ibeji, Agemo, and Freyja in-vehicle stack blueprint image workflow 

### PR DESCRIPTION
**Pull request by ladatz**
These github workflows checks out the eclipse-chariott/Agemo repository  and eclipse-ibeji/ibeji repository and builds images with the configuration and settings needed to run in the integrated mode with the other workloads in the software-orchestration blueprint. It then pushes the resulting images to the blueprint GitHub container registry. The resulting images include:

- eclipse-agemo pub-sub-service
- An eclipse mosquitto mqtt broker using a custom config with agemo
- eclipse-ibeji invehicle digital twin service
- eclipse-freyja a digital twin cloud-synchronization application with a mocked cloud component and integration with ibeji

### CODE REVIEW

**Review by ashbeitz — APPROVED** [review]
Approved

**Comment by ladatz** [comment]
@eriksven Can we please "squash and merge" this one once it has been reviewed? I ended up with a lot of commits here. I'll make sure the next one is cleaner

File: .github/workflows/containerize-invehicle-stack-agemo.yml
Comment (eriksven) [inline .github/workflows/containerize-invehicle-stack-agemo.yml:1]: The specified license in the LICENSE file in the root directory of this repository is Apache 2.0. Do you really want to license the workflows under another license here? 

In addition, the Eclipse Project handbook proposes to include a full copyright header in addition to the License Identifier. See #16 for more details.

**Comment by eriksven** [comment]
The workflows look good to me. I added a comment to add a full copyright header. Also there is a merge conflict with the chariott workflow. Can you resolve this? 

For the merge I will use "squash and merge" this time.

File: .github/workflows/containerize-invehicle-stack-agemo.yml
Comment (ladatz) [reply .github/workflows/containerize-invehicle-stack-agemo.yml:1]: I am waiting for some confirmation from our side about the license question

**Comment by ladatz** [comment]
> The workflows look good to me. I added a comment to add a full copyright header. Also there is a merge conflict with the chariott workflow. Can you resolve this?
> 
> For the merge I will use "squash and merge" this time.

I am looking for some confirmation from my team about the license question. I also cleaned up the history of this PR and added the Freyja workflow as well.

File: .github/workflows/containerize-invehicle-stack-agemo.yml
Comment (inf17101) [inline .github/workflows/containerize-invehicle-stack-agemo.yml:42]: I am thinking about, shall we define the version tag one times e.g. as Github action env variable to ensure to reduce possibility of making mistakes. Someone might forget to adjust the image version tag when checking out another agemo version. To define this once would keep it automatically in sync. We could do this for all image builds as well, if you like the idea.

File: .github/workflows/containerize-sample-workloads.yaml.yml
Comment (inf17101) [inline .github/workflows/containerize-sample-workloads.yaml.yml:1]: Why is it .yaml.yml?

File: .github/workflows/containerize-invehicle-stack-agemo.yml
Comment (inf17101) [reply .github/workflows/containerize-invehicle-stack-agemo.yml]: I have seen that you have accepted it and pushed it, but unfortunately I am not able to close the conversation.

File: .github/workflows/containerize-sample-workloads.yaml.yml
Comment (ladatz) [reply .github/workflows/containerize-sample-workloads.yaml.yml:1]: This was a typo and the file has been removed from this PR altogether (that workflow will go in with the sample code).

File: .github/workflows/containerize-invehicle-stack-agemo.yml
Comment (eriksven) [reply .github/workflows/containerize-invehicle-stack-agemo.yml:1]: @ladatz any updates on the licensing topics here?

Comment (ladatz) [reply .github/workflows/containerize-invehicle-stack-agemo.yml:1]: @eriksven As far as I know, we are still waiting on our open source office team which is reviewing contribution process. I heard we had made some progress and I will update this once I have the green light

Comment (ladatz) [reply .github/workflows/containerize-invehicle-stack-agemo.yml]: @eriksven We are good to merge this. I have updated the license to Apache 2, to be consistent with the rest of the blueprint

**Comment by eriksven** [comment]
@ladatz I missed to see that you already updated the license headers in this PR and merged #20 before now. Most likely as a result, there is now a merge conflict with the containerize-invehicle-stack-chariott.yml file. Can you resolve this?

**Comment by ladatz** [comment]
> @ladatz I missed to see that you already updated the license headers in this PR and merged #20 before now. Most likely as a result, there is now a merge conflict with the containerize-invehicle-stack-chariott.yml file. Can you resolve this?



> @ladatz I missed to see that you already updated the license headers in this PR and merged #20 before now. Most likely as a result, there is now a merge conflict with the containerize-invehicle-stack-chariott.yml file. Can you resolve this?

@eriksven No problem, I have resolved the merge conflict now.

**Pull request merged**
